### PR TITLE
Eq 1394 escape char fix

### LINKED ISSUE #1394: Entering single/double quote in text field when revisited results in uni-code
### Expected behaviour
Must display the text as entered by respondent

### Actual behaviour
Displays text with unicode in place of single/double quotes (PFA)

### Steps to reproduce the behaviour
Launch any survey with text field
Navigate to text field and enter text with single/double quote and click save and continue
press <previous>

**NOTE: Defect exists in PROD**

### Screenshot
screen shot 2017-12-05 at 12 11 28

### PR DESCRIPTION
**Pull request by yoganandkunche**
### What is the context of this PR?
Fixes #1394 

### How to review 
Retest #1394

### CODE REVIEW

File: tests/functional/spec/textfield.spec.js [hunk 1]
```diff
@@ -10,4 +11,14 @@ describe('Textfield', function() {
         .hasFocus(TextFieldPage.answer()).should.eventually.be.true;
       });
     });
+
+  it('Given a text entered in textfield , when user submits and revisits the textfield, then the textfield must contain the text entered previously', function() {
```
Comment (shahi645) [inline tests/functional/spec/textfield.spec.js]: Should capitalise the `When` and `Then`

**Review by dwyeradam — APPROVED** [review]
I'm less fussy about When Then but good to keep standards.

**Comment by dwyeradam** [comment]
Worth adding the same test to TextArea as well as just TextField.

File: tests/functional/spec/textfield.spec.js [hunk 2]
```diff
@@ -10,4 +11,14 @@ describe('Textfield', function() {
...
+    return helpers.openQuestionnaire('test_textfield.json').then(() => {
+      return browser
+        .setValue(TextFieldPage.answer(), "'Twenty><&Five'")
+        .click(TextFieldPage.submit())
+        .click(SummaryPage.answerEdit())
```
Comment (jonnyshaw89) [inline tests/functional/spec/textfield.spec.js]: Is it worth checking that the value is correct on the Summary page?

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Review by shahi645 — APPROVED** [review]
Fixed 👍

**Review by jonnyshaw89 — APPROVED** [review]

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]
You need to change to commit message to be more descriptive and contain the issue number

**Review by jonnyshaw89 — APPROVED** [review]

**Pull request merged**